README.md

```diff
@@ -114,27 +122,28 @@ The name used to reference the client organization.
 
 Data used to hydrate the Utility Connect front-end and interface with the API.
 
-Expect data in this shape for the `create` scope:
+#### `create` scope
+
+Data is expected in the following format for the `create` scope:
 
 ```javascript
 {
   user: {
     email: 'fake_email@example.com', // user's email
-    firstName: 'Falsey', // user's first name
-    lastName: 'Farcicle' // user's last name
+    firstName: 'First', // user's first name
+    lastName: 'Name' // user's last name
```

Comment:
also if you are gonna change this, this should be `'Last`

Reply:
oops hahaha thats what I MEANT